### PR TITLE
updated for dummy model and ERA5 land

### PR DESCRIPTION
**Pull request by vgro**
Here, I updated the documentation for climate data downloading and pre-processing once more to make it work for our dummy model as discussed on Tuesday.

The grid is 9x9 cells of 90 m (which corresponds to 3 arcmin) with x/y coordinates and I added a `time_index` coordinate to test the looping over time in the `abiotic_simple_model`.

### CODE REVIEW

File: docs/source/data_recipes/CDS_toolbox_template.md [hunk 1]
```diff
@@ -26,76 +26,107 @@ At present, the pre-processing does not include scaling or topographic adjustmen
...
-* Air humidity (typically 2m; relative or specific humidity)
-* Air pressure (typically mean sealevel or surface pressure)
-* Wind speed (typically 10m)
+* Relative humidity (typically 2m; relative or specific humidity)
+* Atmospheric pressure (typically mean sealevel or surface pressure)
 * Precipitation
-* Top of atmosphere short-wave downward radiation
-* CO2 concentration (for future projections)
-* optional: soil temperature and soil moisture for initialisation
+* optional: atmospheric $\ce{CO_{2}}$ concentration, soil temperature, soil moisture
```
Comment (jacobcook1995) [inline docs/source/data_recipes/CDS_toolbox_template.md:44]: I would maybe split this into a separate paragraph bullet point list, as find it a bit a weird to have optional inputs listed in a bullet point list of the required variables

File: docs/source/data_recipes/ERA5_preprocessing_example.md [hunk 2]
```diff
@@ -100,56 +97,78 @@ variables according to the Virtual Rainforest naming convention (see
...
+of 4.5 km has to be added.
+
+```{code-cell} ipython3
+dataset_xy = dataset_renamed.rename_dims({'longitude':'x','latitude':'y'}).assign_coords({'x':np.arange(0,180000,9000),'y':np.arange(0,180000,9000)}).drop({'longitude','latitude'})
+dataset_xy.coords
+```
+
+### 8. Scale to 90 m resolution
+
+The Virtual Rainforest run on a 90 x 90 m grid. This means that some form of spatial
```
Comment (jacobcook1995) [inline docs/source/data_recipes/ERA5_preprocessing_example.md:142]: "is run" or "will be run"?

**Review by jacobcook1995 — APPROVED** [review]
LGTM, (though admittedly I have no experience of working with this type of data)

File: docs/source/data_recipes/ERA5_preprocessing_example.md [hunk 3]
```diff
@@ -14,31 +14,29 @@ kernelspec:
   name: vr_python3
 ---
 
-# Simple climate data pre-processing example for dummy module
+# Simple climate data pre-processing example for dummy model
 
 This section illustrates how to perform simple manipulations to adjust ERA5 data to use
-in the Virtual Rainforest. This includes reading climate data from netcdf, converting
+in the Virtual Rainforest. This includes reading climate data from grib, converting
 the data into an input formate that is suitable for the abiotic module (e.g. Kelvin to
```
Comment (davidorme) [inline docs/source/data_recipes/ERA5_preprocessing_example.md:21]:
```suggestion
the data into an input format that is suitable for the abiotic module (e.g. Kelvin to
```

File: docs/source/data_recipes/ERA5_preprocessing_example.md [hunk 4]
```diff
@@ -100,56 +97,78 @@ variables according to the Virtual Rainforest naming convention (see
...
+
+```{code-cell} ipython3
+dataset_xy = dataset_renamed.rename_dims({'longitude':'x','latitude':'y'}).assign_coords({'x':np.arange(0,180000,9000),'y':np.arange(0,180000,9000)}).drop({'longitude','latitude'})
+dataset_xy.coords
+```
+
+### 8. Scale to 90 m resolution
+
+The Virtual Rainforest is run on a 90 x 90 m grid. This means that some form of spatial
+scaling has to be applied to the dataset, **which is not yet implemented!**
```
Comment (davidorme) [inline docs/source/data_recipes/ERA5_preprocessing_example.md:143]: Not strictly true - these data will run it at that resolution!

File: docs/source/data_recipes/ERA5_preprocessing_example.md [hunk 5]
```diff
@@ -100,56 +97,78 @@ variables according to the Virtual Rainforest naming convention (see
...
+The dummy model iterates over time indices rather than real datetime. Therefore, we add
+a `time_index` coordinate to the dataset:
+
+```{code-cell} ipython3
+dataset_xy_timeindex = dataset_xy_dummy.rename_dims({'time':'time_index'}).assign_coords({'time_index':np.arange(0,24,1)})
+dataset_xy_timeindex.coords
 ```
 
-### 6. Save netcdf
+### 10. Save netcdf
```
Comment (davidorme) [inline docs/source/data_recipes/ERA5_preprocessing_example.md:167]: I'd add a further section giving the grid config `TOML` to load this data, which can link back to your comment about the offset.

```toml
[core.grid]
cell_nx = 9
cell_ny = 9
cell_area = 8100
xoff = -45.0
yoff = -45.0
```

**Review by davidorme — APPROVED** [review]
LGTM - minor comments

File: docs/source/data_recipes/ERA5_preprocessing_example.md (hunk 4 above)
Comment (vgro) [reply docs/source/data_recipes/ERA5_preprocessing_example.md:143]: I rephrased it to make this clearer

File: docs/source/data_recipes/ERA5_preprocessing_example.md (hunk 5 above)
Comment (vgro) [reply docs/source/data_recipes/ERA5_preprocessing_example.md:167]: I added a section at the end and referred to it where I mention the offset. I also added a link to the config section in the VR setup.

**Pull request merged**